cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -46,7 +47,7 @@ const (
 	DefaultImageRepository = "gcr.io/google_containers"
 )
 
-func addDefaultingFuncs(scheme *runtime.Scheme) error {
+func addDefaultingFuncs(scheme *kruntime.Scheme) error {
```

Comment:
nit: if std "runtime" not used in this file, do we need now to rename that import ?

Reply:
yup, runtime had been in there, but is no longer, updating